### PR TITLE
NL2LTL Integration

### PR DESCRIPTION
**Pull request by TathagataChakraborti**

### CODE REVIEW

File: .gitignore
Comment (francescofuggitti) [inline .gitignore:1]: @TathagataChakraborti, why ignore it?

**Comment by TathagataChakraborti** [comment]
@francescofuggitti @jkeskingvillage i am ready to merge

**Comment by jkeskingvillage** [comment]
> @francescofuggitti @jkeskingvillage i am ready to merge

Good!

**Review by francescofuggitti — APPROVED** [review]
Looks good!

**Pull request merged**